app/src/main/java/com/strv/movies/MainActivity.kt

```diff
@@ -46,13 +44,94 @@ class MainActivity : ComponentActivity() {
                                 DarkLightModeSwitchIcon(isDarkTheme = isDarkTheme)
                             }
                         )
+                        Greeting("MOVIES")
+                        LogInMoviesApp()
                         MoviesList(movies = OfflineMoviesProvider.getMovies())
                     }
                 }
             }
         }
     }
 
+    @OptIn(ExperimentalFoundationApi::class)
+    @Composable
+    fun LogInMoviesApp() {
+        Column(
+            horizontalAlignment = Alignment.CenterHorizontally,
+            modifier = Modifier
+                .padding(start = 40.dp, bottom = 10.dp, top = 18.dp, end = 20.dp)
+                .width(300.dp)
+        ) {
+            LogInElements()
+            otherElementsForLogin()
+        }
+    }
+
+    @Composable
+    private fun otherElementsForLogin() {
+        Button(
+            onClick = {},
+            modifier = Modifier
+                .width(280.dp)
```

Comment:
It would make sense to use the `.fillMaxWidth()` also here, so the button width is adapted to screen size. The space between button and edge of screen will be preserved because you used padding for start and end in parent `Column`.

Reply:
done